katello/katello/backup.rb

```diff
@@ -336,6 +343,14 @@ def setup_opt_parser
           end
         end
 
+        opts.on("--split-pulp-tar SIZE", "Use split on pulp_data.tar to make smaller files") do |size|
+          @options[:split_pulp_tar] = size
+        end
+
+        opts.on("--split-pulp-tar-rotate-script /PATH/SCRIPT", "Use this script to facilitate splitting pulp_data.tar") do |script|
+          @options[:split_pulp_tar_script] = script
+        end
```

Comment:
This looks like its optional - if that is correct, could you make a note saying so? i.e.
`Use this script to facilitate splitting pulp_data.tar. This is optional, a default script is provided`

Reply:
I can re-write that bit.